regenerate_and_fix_sdk.py

```diff
@@ -0,0 +1,122 @@
+import os, sys, subprocess, re
+from distutils.dir_util import copy_tree, remove_tree
+
+# Need python 3.6 for new subprocess API
+MIN_PYTHON = (3, 6)
+if sys.version_info < MIN_PYTHON:
+    sys.exit("Python %s.%s or later is required.\n" % MIN_PYTHON)
+
+# Configuration
+verbose = True
+autorest_target_dir = ".%stmp" % os.sep # Default (out-of-tree) target directory is ./tmp 
+swagger_source_branch = "keyvault_preview" # Source branch in swagger repo to generate from
+autorest_tag = "package-7.0-preview" # Autorest tag to use, if any
+run_autorest = True # If false, assumes that we just need to repair generated code that is already in tree!
+fix_generated_code = True # if false, assumes that we just need to run autorest
+source_rest_spec = "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/%s/specification/keyvault/data-plane/readme.md" % swagger_source_branch # Builds the URL to Swagger readme from branch
+autorest_cmd = "autorest --java %s --azure-libraries-for-java-folder=%s" % (source_rest_spec, autorest_target_dir)
+
+if autorest_tag is not None:
+    autorest_cmd += " --tag=%s" % autorest_tag
+
+# Constant values
+GENERATED_JWK_MODEL_FILES = [
+    'azure-keyvault/models/JsonWebKey.java', 
+    'azure-keyvault/models/JsonWebKeyEncryptionAlgorithm.java', 
+    'azure-keyvault/models/JsonWebKeyOperation.java', 
+    'azure-keyvault/models/JsonWebKeySignatureAlgorithm.java',
+    'azure-keyvault/models/JsonWebKeyType.java'
+] # list of JWK models which will be generated by autorest that we will need to delete
+
+CODE_TARGET_DIRECTORY = os.path.join(".", "azure-keyvault", "src", "main", "java", "com", "microsoft", "azure", "keyvault")
```

Comment:
I would put this script at the base of the repo then use the script location as the base rather than relying on the current directory being set properly